hcatalog/webhcat/svr/src/main/java/org/apache/hive/hcatalog/templeton/Main.java

```diff
@@ -306,18 +308,30 @@ private Connector createChannelConnector(Server server) {
   public FilterHolder makeAuthFilter() throws IOException {
     FilterHolder authFilter = new FilterHolder(AuthFilter.class);
     UserNameHandler.allowAnonymous(authFilter);
+  
+    // compatible with Hadoop 3.3.x.
+    // https://issues.apache.org/jira/browse/HIVE-24083
```

Comment:
Please remove the link to the JIRA. We will pre-pend it to the commit message so developers will be able to find it.

Reply:
Sure. I'll remove the link